ci/docker.sh

```diff
@@ -83,9 +79,16 @@ if [ -n "${docker_tag_full:-}" ]; then
     publish_tags=("${docker_tag_full}")
   fi
 
+  # Append -relay to tag names for relay chain images
+  if [[ "${image_artifact}" == *"relay"* ]]; then
```

Comment:
The idea is to catch ANYTHING that contains `relay` and append -relay to it? That means string like this will match as well `no_relay_here`. Just curious if we need smth more strict or not really?

Reply:
It can be an exact match on ""zkverify-relay"